rewriter/AttrReader.cc

```diff
@@ -39,6 +39,16 @@ pair<core::NameRef, core::LocOffsets> getName(core::MutableContext ctx, ast::Exp
                 res = core::Names::empty();
             }
             loc = lit->loc;
+            if (::sorbet::debug_mode) {
```

Comment:
Why are we casing on debug mode here?

Reply:
Ah I see, this is to support some ENFORCE. Usually we use the DEBOG_ONLY macro for this.